### DISCUSSION TITLE
C# wasm

### DISCUSSION THREAD

**Discussion by bosoni** [rokups/rbfx#464] [Q&A]
Hi. 
C# bindings are awesome, because I like C#.

But is there some way to "packet" C# scripts with rbfx and compile it to wasm?  (I mean like (maybe) godot does, it uses mono-wasm or something and running C# IL under web (yeah not the fastest because VM inside VM)).

**Comment by gleblebedev** [comment]
The plan is to move to .net6 on all platforms and then use Blazor for web.
